tools/azure-sdk-tools/devtools_testutils/sanitizers.py

```diff
@@ -283,4 +475,25 @@ def _send_sanitizer_request(sanitizer: str, parameters: dict) -> None:
         headers={"x-abstraction-identifier": sanitizer, "Content-Type": "application/json"},
         json=parameters
     )
+    breakpoint()
+    response.raise_for_status()
+
+
+def _send_transform_request(transform: str, parameters: dict) -> None:
+    """Sends a POST request to the test proxy endpoint to register the specified transform.
+
+    If live tests are being run, no request will be sent.
+
+    :param str transform: The name of the transform to add.
+    :param dict parameters: The transform constructor parameters, as a dictionary.
+    """
+
+    if is_live():
+        return
+
+    response = requests.post(
+        f"{PROXY_URL}/Admin/AddTransform",
+        headers={"x-abstraction-identifier": transform, "Content-Type": "application/json"},
```

Comment:
So this one _looks_ to me to only add `session` level sanitizer. However, further up in `set_reset_request` we actually always send for a recordingId (unless it gets called and the fixture can't find the active recordingId).

Can we have a quick chat about the PR?